src/modules/meteor/index.js

```diff
@@ -50,12 +50,12 @@ export function setup(api) {
     if (config.ssl.upload !== false) {
       list.copy('Copying SSL Certificate Bundle', {
         src: resolvePath(basePath, config.ssl.crt),
-        dest: '/opt/' + config.name + '/config/bundle.crt'
+        dest: '/opt/' + config.name + '/config/certs/bundle.crt'
```

Comment:
This is a breaking change. It would require `mup setup` to be run for ssl to work again. You should be able to use the same path.